### ISSUE TITLE
SP5: pie labels alignment

### ISSUE THREAD

**Issue by jaguarxii** [ScottPlot/ScottPlot#4216]
At rendering a pie chart in a rectangle, pieslice labels missalign from pie

![image](https://github.com/user-attachments/assets/c9744ce0-1346-418c-9803-37df5369018f)

ScottPlot 5.0.38

**Code Sample:** See http://scottplot.net/faq/repro/ for tips about creating reproducible code samples

```cs
Random randNum = new Random();
        double[] sampleData = Enumerable
            .Repeat(0, 9)
            .Select(i => Convert.ToDouble(randNum.Next(100, 1000)) / 100)
            .ToArray();

        ScottPlot.Plot myPlot = new();
        List<PieSlice> tmpPie = new List<PieSlice>();
        PieSlice tmpSlice = null;
        IPalette palette = new ScottPlot.Palettes.Category10();

        myPlot.Title("Scott Pie");

        myPlot.Axes.Title.Label.Text = "Scott Pie";
        myPlot.Axes.Title.Label.FontSize = 32;
        myPlot.Axes.Title.Label.Bold = false;

        myPlot.Layout.Frameless();
        myPlot.HideGrid();

        foreach (double item in sampleData)
        {
            tmpSlice = new()
            {
                Value = item,
                FillColor = palette.GetColor((int)randNum.Next(10)),
                Label = "Lab\r\n" + item.ToString()
            };

            tmpPie.Add(tmpSlice);
        }

        Pie myPie = myPlot.Add.Pie(tmpPie);
        myPie.DonutFraction = 0.6;
        myPie.ShowSliceLabels = true;
        myPie.LineColor = ScottPlot.Color.FromSKColor(SkiaSharp.SKColors.Transparent);

        myPlot.ShowLegend(Edge.Bottom);

        return myPlot.GetSvgXml(800, 450);
```

**Comment by CoderPM2011** [comment]
This has been fixed in #4211, but not released yet.

This is caused by the different proportions of the X and Y axes.
You can fix it using `myPlot.Axes.SquareUnits();`
reference: https://scottplot.net/cookbook/5.0/AdvancedAxis/SquareAxisUnits/

**Comment by jaguarxii** [comment]
Thanks it works as described.

![image](https://github.com/user-attachments/assets/6e5fc8e5-c171-47e5-8ff9-167027b2fc2b)
